test/fixtures/external_api_clients.yml

```diff
@@ -309,4 +309,50 @@ mailchimp_plugin:
     end
 
     # at the end of the file we have to implicitly return the class 
-    SyncToMailchimp
\ No newline at end of file
+    SyncToMailchimp
+
+sync_attribute_to_api_namespace_plugin:
+  api_namespace: one
+  slug: sync-attribute-to-api-namespace
+  label: SyncAttributeToApiNamespace
+  enabled: true
+  metadata: {
+            'attribute_name': 'test_attribute',
```

Comment:
@alis-khadka what's the API surface here? You will need to create some docs like we did for the Mailchimp plugin: https://github.com/restarone/violet_rails/issues/720#issuecomment-1183585540